bookbotproject/conversationtree/serializers2.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import serializers
```

Comment:
Serializers2 can be in the same file with serializer, no need for 2 separate files. Just add the classes under the existing ones